app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Ingredients</h1>
+
+<% @ingredients.each do |ingredient| %>
+  <h3><%= ingredient.name %></h3>
```

Comment:
This looks good. I might just write it out the way the US gave an example to display it:
```
As a visitor,
When I visit '/ingredients'
I see a list of all the ingredients including their name and cost
(e.g. "Ground Beef: 2"
          "Salt: 4")
```
So it would probably look like:
```ruby
<% @ingredients.each do |ingredient| %>
  <p><%= ingredient.name %>: <%= ingredient.cost %></p>
<% end %>
```